src/controllers/user/artists/routes/{id}/index.js

```diff
@@ -168,8 +168,8 @@ module.exports = function () {
           {
             model: User,
             required: false,
-            attributes: ['id', 'displayName'],
-            as: 'user'
+            attributes: ['id', 'displayName']
+            // as: 'user'
```

Comment:
```suggestion
```

This can just be deleted, there's no reason to keep it, as it's in the git history :)